fiboa_cli/datasets/es_nc.py

```diff
@@ -0,0 +1,68 @@
+import re
+
+import requests
+from os import path, makedirs
+import pandas as pd
+
+from .es import ESBaseConverter
+from ..util import log
+
+
+# SOURCES = "https://filescartografia.navarra.es/2_CARTOGRAFIA_TEMATICA/2_6_SIGPAC/" # FULL Download timeout
+class NCConverter(ESBaseConverter):
+    id = "es_nc"
+    short_name = "Spain Navarra"
+    title = "Spain Navarra Crop fields"
+    description = """
+    SIGPAC Crop fields of Spain - Navarra
+    """
+    license = "CC-BY-4.0"  # https://sigpac.navarra.es/descargas/
+    attribution = "Comunidad Foral de Navarra"
+    providers = [
+        {
+            "name": "Comunidad Foral de Navarra",
+            "url": "https://gobiernoabierto.navarra.es/",
+            "roles": ["producer", "licensor"]
+        }
+    ]
+    columns = {
+        "id": "id",
+        "geometry": "geometry",
+        "BEGINLIFE": "determination_datetime",
+        "IDUSO24": "crop:code",
+        "crop:name": "crop:name",
+        "crop:name_en": "crop:name_en",
+    }
+    column_migrations = {
+        "BEGINLIFE": lambda col: pd.to_datetime(col, format="%d/%m/%Y"),
+    }
+    use_code_attribute = "IDUSO24"
+    index_as_id = True
+
+    def get_urls(self, cache=None):
```

Comment:
A method called get_ that is downloading and writing something might not be ideal. Why is this done here?

Reply:
Valid point, separating will make the code more readable...